src/staging/src/pages/active-advisories.js

```diff
@@ -161,10 +161,10 @@ const PublicActiveAdvisoriesPage = ({ data }) => {
 
     // exclude unpublished parks
     let q =
-      "/public-advisories/count?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true"
+      "/public-advisories/count"
 
     if (advisoryType !== "all") {
-      q += `&eventType.eventType_contains=${advisoryType}`
+      q += `?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true&eventType.eventType_contains=${advisoryType}`
     }
```

Comment:
Is this logic correct? It seems like you've removed the filter for the "all" advisory type, so it will include unpublished advisories and advisories from undisplayed protected areas?

Reply:
I'm not sure why, but without filtering has the correct number of counts
- Without filtering, `https://test-cms.bcparks.ca/public-advisories/count?` has 455 which is correct as you can see [here](https://test-cms.bcparks.ca/admin/plugins/content-manager/collectionType/application::public-advisory.public-advisory?page=1&pageSize=10&_sort=title:ASC)
- With filtering, `https://test-cms.bcparks.ca/public-advisories/count?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true` has 550 which is incorrect
- With filtering, and additional filter or query, `https://test-cms.bcparks.ca/public-advisories/count?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true&eventType.eventType_contains=anchorage` has 3 which is correct as you can see [here](https://test-cms.bcparks.ca/admin/plugins/content-manager/collectionType/application::public-advisory.public-advisory?page=1&pageSize=10&_sort=eventType.eventType:ASC&_where[0][eventType.eventType]=Anchorage)

...but I couldn't find why the counter with filtering doesn't work properly 🥲 Please let me know if you find anything.

Reply:
The root cause is this Strapi bug...
https://github.com/strapi/strapi/issues/7631
Right now, we don't have any protectedAreas with isDisplayed = false (or any unpublished protected areas), so the code is working with no filter.  But if we had public advisories for these protectedAreas, then it would return the wrong count. 

I think we have 2 options:
1. Remove the deep filtering (as you have done) and use the unfiltered count.
2. Add a custom count endpoint to the protected-areas controller that uses a knex subquery to get the correct count.

Option 1 is only an issue if we have unpublished protectedAreas that also have publicAdvisories.

Option 2 fixes the issue for Strapi 3, but then we are adding a custom endpoint that probably won't be needed for Strapi 4.